### PR TITLE
Return senders from /v0/api/getSenders endpoint

### PR DESCRIPTION
**Pull request by nhynes**
Example req/res:
```
❯ curl -i http://localhost:1234/v0/api/getSenders -H 'x-oasis-insecure-auth: 1' -H 'x-oasis-session-k
ey: whatever'
HTTP/1.1 200 OK
Vary: Origin
X-Oasis-Trace-Id: -1
Date: Fri, 30 Oct 2020 11:35:04 GMT
Content-Length: 61
Content-Type: text/plain; charset=utf-8

{"addresses":["0xB8b3666d8fEa887D97Ab54f571B8E5020c5c8b58"]}
```

This endpoint will be used when creating identities with the gateway's senders as trusted.

### CODE REVIEW

File: api/v0/info/entity.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package info
+
+// GetVersionResponse is the response to the health request.
+type GetVersionResponse struct {
+	Version int `json:"version"`
+}
```
Comment (nhynes) [inline api/v0/info/entity.go:6]: this isn't new. I just renamed the `version` package to `info`.

**Review by mitjat — APPROVED** [review]
Thank you!

For operationalizing this: I remember we used to run out of funds in our gateway wallet fairly frequently, until we switched to directly using "the ur-wallet" that was previously only used to periodically fund the dedicated gateway wallet.
Once we switch to multiple wallets, will we have to be mindful about keeping them well funded?

**Comment by nhynes** [comment]
>  directly using "the ur-wallet"

I was hoping we could use several of these. `0xfffff.... >> 4` is still effectively infinity, though.

File: api/v0/info/handler.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+// GetVersion returns the version of the component
+func (h InfoHandler) GetVersion(ctx context.Context, v interface{}) (interface{}, error) {
+	return &GetVersionResponse{
+		Version: 0,
+	}, nil
+}
+
+// GetVersion returns the version of the component
```
Comment (nhynes) [inline api/v0/info/handler.go:50]:
```suggestion
// GetSenders returns the addresses of the accounts the gateway uses
// to sign transactions.
```

**Pull request merged**